### PR TITLE
Files added: Assignment 5,6

### PR DESCRIPTION
**Pull request by SamikshaWani**

### CODE REVIEW

**Review by Oshin-250197 — APPROVED** [review]

**Review by roshnidurugwar — APPROVED** [review]

**Review by yogesh141293 — APPROVED** [review]

File: Samiksha/html/Assignment4/Assig4_2.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Question 2</title>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Samiksha/html/Assignment4/Assig4_2.html]: add space after comma always like table, th, td {}

File: Samiksha/html/Assignment4/Assign4_1.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Question 1</title>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Samiksha/html/Assignment4/Assign4_1.html]: add space after comma

File: Samiksha/html/Assignment4/Assign4_3.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Question 3</title>
+        <style>
+            table,th,td {
```
Comment (SnehaMakade) [inline Samiksha/html/Assignment4/Assign4_3.html]: add space after comma

File: Samiksha/html/Assignment4/Assign4_3.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                border:1px solid black;
```
Comment (SnehaMakade) [inline Samiksha/html/Assignment4/Assign4_3.html]: add space after colon always

File: Samiksha/html/Assignment6/Assign6_1.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        <link rel="stylesheet" href="Styles.css" />
+    </head>
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>
+        <br />
+        <hr />
+        <h2><i>Personal Details</i></h2>
+        <form>
+            <label for="fname"><b>First Name:</b></label>
+            <input type="text" id="fname" name="fname" placeholder="First Name" /><br /><br />
```
Comment (SnehaMakade) [inline Samiksha/html/Assignment6/Assign6_1.html]: placeholders second words 1st letter should be small make this change in all input boxes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
make the changes as per comments given

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**